i18n-helpers/fuzz/Cargo.toml

```diff
@@ -7,6 +7,11 @@ publish = false
 [package.metadata]
 cargo-fuzz = true
 
+[lints.clippy]
```

Comment:
This does not work because `fuzz` dir is not a part of the workspace but we can make it part of the workspace once a new version of `cargo-fuzz` released which contains https://github.com/rust-fuzz/cargo-fuzz/pull/357

Reply:
Nice! I first considered making the fuzz tests part of the workspace, but it seemed easier to leave it like this. It's cool to hear that we'll soon be able to merge everything.